sample_tracking/.firebaserc

```diff
@@ -1,5 +1,8 @@
 {
   "projects": {
-    "default": "river-sky-386919"
-  }
-}
+    "default": "timberid-prd",
```

Comment:
I dont understand why the default is prd.  does this mean when you do deploy it by default goes to PRD?  that sounds dangerous
we want deploy to PRD to only happen if its intentional.